index.js

```diff
@@ -89,15 +98,15 @@ function CogLvlCell(cogLvl) {
 function CogLvlColumn() {
   let arr = [];
   // TODO For lvl 13+ cogs
-  for (let cogLvl = 12; cogLvl >= 1; cogLvl--) {
+  for (let cogLvl = 20; cogLvl >= 1; cogLvl--) {
```

Comment:
This is where we'd conditionally specify iterating through levels 20 to 1 for TTR, and levels 12 to 1 for classic Toontown. For example:
```suggestion
  const maxCogLvl = state.game === 'ttr' ? 20 : 12;
  for (let cogLvl = maxCogLvl; cogLvl >= 1; cogLvl--) {
```

Reply:
change implemented.